webrecorder/browsertrix
# feat: Rename org when accepting org invite for first admin

Resolves https://github.com/webrecorder/browsertrix/issues/1874


### Manual testing
1. Log in as super user
2. Create a new org or choose an org without any users, or remove users by setting everyone to "Viewer" and then removing them
3. Update org name to be its UUID
4. Invite a new user as an admin, using an email you have access to
5. Copy-paste the invite link path in your inbox. Verify you see the "Welcome to Browsertrix" sign up form
6. Click "Next". Verify you see the org rename form with the name prefilled to `{your name}'s Archive`
7. Click "Go to Dashboard". Verify you're taken to the correct page
8. Repeat steps 2-7 for an existing user.

Regression test the following:
1. New and existing user invited after first admin
2. New and existing user invited as first admin when org name is not set to its UUID
3. Update app name and slug in org settings

### Screenshots

#### New User Invite - Step 1
<img width="956" alt="Screenshot 2024-06-25 at 4 16 47 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/7e9df551-852a-49cb-8750-baaff3cd65ce">

#### New User Invite - Step 2
<img width="983" alt="Screenshot 2024-06-25 at 4 17 03 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/fb2aa03c-b352-4bf4-ae4a-2c5447cc51bb">

#### Existing User Invite - Step 1
<img width="976" alt="Screenshot 2024-06-25 at 4 18 46 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/d9303a8c-9d8a-4b02-92d7-4d19abc0b3b8">

#### Existing User Invite - Step 2
<img width="951" alt="Screenshot 2024-06-25 at 4 41 36 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/c95b5b81-cd69-419d-850d-6630cbcedebe">

#### Taken org name error
<img width="506" alt="Screenshot 2024-06-25 at 4 41 48 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/5927a6f4-b908-4f6d-8cb1-a75499279bc8">

### Invalid invite error
<img width="490" alt="Screenshot 2024-06-25 at 5 54 49 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/01e2e85a-ae58-49b9-bd35-3c0b55b8d3e0">
